docs/configuration.md

```diff
@@ -4,8 +4,10 @@ certain default messages and paths, enable or disable features, set refresh rate
 properly or not function at all when not configured through the properties file. 
 
 ## Quickstart
-To configure the framework, create a config.properties file in the root directory of your project. If no config.properties is found on startup,
-a default configuration will be used. Note that properties are validated in runtime, so properties of unused agents don't need to be configured 
+To configure the framework, create a config.properties file in the root of the resources directory of your project.   

```

Comment:
Dit moet dan toch zijn in "src/main/resources/" in plaats van root?

Reply:
"in the root of the resources directory", ik zal het even bold maken